src/renderer/i18n/nl.js

```diff
@@ -0,0 +1,483 @@
+      },
+      defaultLayer: {
+        label: "Default laag",
+        noDefault: "Geen default",
+        help: `De default laag waarmee het toetsenbord zal opstarten.`,
+      },
+      factoryReset: {
+        button: "Reset EEPROM naar fabrieksinstellingen",
+        dialog: {
+          title: "Reset EEPROM naar fabrieksinstellingen",
+          contents: `Dit zal de EEPROM resetten naar de fabrieksinstellingen. Je zal al je aanpassingen verliezen.`,
+        },
+      },
+      flash: {
+        preferExternal: {
+          label: "Gebruik een extern programma om te flashen",
+        },
+      },
+      led: {
+        label: "LED voorkeuren",
+        idle: {
+          label: "Inactieve tijd voordat LEDs afgezet worden",
+          help: `Tenzij dit uitgezet is, zullen de LEDs uitgaan na een ingestelde tijd.`,
+          disabled: "Uit",
+          oneMinute: "1 minuut",
+          twoMinutes: "2 minuten",
+          threeMinutes: "3 minuten",
+          fourMinutes: "4 minuten",
+          fiveMinutes: "5 minuten",
+          tenMinutes: "10 minuten",
+          fifteenMinutes: "15 minuten",
+          twentyMinutes: "20 minuten",
+          thirtyMinutes: "30 minuten",
+          sixtyMinutes: "60 minuten",
+        },
+        brightness: {
+          label: "LED Helderheid",
+          help: "Pas de helderheid van de LEDs op het toetsenbord aan.",
+        },
+        default: {
+          autoSave: {
+            label: "Zet het automatisch opslaan van de default LED mode aan",
+            help: `Wanneer dit is aangezet, wordt de LED mode automatisch als default opgeslaan elke keer ze wordt aangepast.`,
+          },
+          label: "Default LED mode",
+          help: "Selecteer de LED mode waarmee het toetsenbord moet opstarten.",
+        },
+      },
+      plugins: {
+        label: "Plugin voorkeuren",
+        escOneShot: {
+          label: `Laat Escape one-shot toetsen annuleren`,
+          help: `Wanneer dit is aangezet zal de "Escape" toets one-shot toetsen annuleren, anders kan een specifiek ingestelde "annuleer" toets dit doen.`,
+        },
+      },
+    },
+  },
+  keyboardSelect: {
+    unknown: "Onbekend",
+    selectPrompt: "Selecteer een toetsenbord:",
+    noDevices: `Geen toetsenborden gevonden.`,
+    connect: "Verbinding maken",
+    disconnect: "Verbinding verbreken",
+    scan: "Zoeken",
+    installUdevRules: "Installeer udev regels",
+    permissionError: `Je computer laat Chrysalis niet spreken met je toetsenbord. (Je hebt geen lees/schrijf rechten naar {{path}}.)`,
+    permissionErrorSuggestion: `Chrysalis kan dit oplossen door een bestand met udev regels te installeren in /etc/udev/rules.d/`,
+  },
+  firmwareUpdate: {
+    calloutTitle: "Belangrijk!",
+    dialog: {
+      selectFirmware: "Selecteer een firmware",
+      firmwareFiles: "Firmware bestanden",
+      allFiles: "Alle bestanden",
+    },
+    options: {
+      onFlash: "Zet terug naar fabrieksinstellengen wanneer geflasht wordt",
```

Comment:
Making this shorter might increase readability.

```suggestion
      onFlash: "Zet terug naar fabrieksinstellengen tijdens flashen",
```

Reply:
and "fabrieksinstell**e**ngen" -> "fabrieksinstell**i**ngen"